javascript/ql/lib/semmle/javascript/PackageExports.qll

```diff
@@ -120,14 +120,21 @@ private DataFlow::Node getAValueExportedByPackage() {
   exists(ImmediatelyInvokedFunctionExpr func, DataFlow::ParameterNode prev, int i |
     prev.getName() = "factory" and
```

Comment:
as you're naming the parameternode below `exports` after its name, could you also rename `prev` to `factory`?